llvm/lib/Support/SHA1.cpp

```diff
@@ -307,3 +307,39 @@ std::array<uint8_t, 20> SHA1::hash(ArrayRef<uint8_t> Data) {
   memcpy(Arr.data(), S.data(), S.size());
   return Arr;
 }
+
+std::string SHA1::hashToString(ArrayRef<uint8_t> Data, StringRef Mapping) {
+  constexpr unsigned BitsPerChar = 6;
```

Comment:
Replace all the `unsigned` by `unsigned int` or `std::size_t`

Reply:
Hi,

Just curious, what is wrong with `unsigned` compared to `unsigned int`? Why the latter is preferred over the former?

Reply:
Just to be more explicit, for example for the Python programmers that could drop by. :-)